core/controllers/question_editor.py

```diff
@@ -99,6 +105,34 @@ def post(self):
             raise self.InvalidInputException(
                 'Skill difficulties must be between 0 and 1')
 
+        html_list = question.question_state_data.get_all_html_content_strings()
+        filenames = (
+            html_cleaner.get_image_filenames_from_html_strings(html_list))
+        image_validation_error_message_suffix = (
+            'Please go to oppia.org/question_editor/%s to edit '
```

Comment:
This is not a valid url, right? Question editor doesn't have a fixed url, depends on if it is opened from the topic or skill editor.

Reply:
done, changed it.